package.json

```diff
@@ -42,6 +42,7 @@
 		"build:web:analyze": "lerna run build:web:analyze --stream",
 		"build:docs": "cd docs && yarn install && yarn build",
 		"changelog": "ts-node -P scripts/changelog/tsconfig.json scripts/changelog/src/index.ts",
+		"fixesmimports": "node scripts/fix_esm_import_paths.js",
```

Comment:
instead of having a script to run to fix esm imports, maybe we use a linter rule import/no-unresolved so that improts must have a file extension or something similar?

Reply:
using eslint is more proactive, I checked few eslint plugins and currently good fit is `require-extensions/require-extensions`,